src/transport/FabricTable.h

```diff
@@ -221,41 +220,40 @@ class DLL_EXPORT AdminPairingInfo
     };
 };
 
-// Once attribute store has persistence implemented, AdminPairingTable shoud be backed using
+// Once attribute store has persistence implemented, FabricTable shoud be backed using
 // attribute store so no need for this Delegate API anymore
-// TODO: Reimplement AdminPairingTable to only have one backing store.
-class DLL_EXPORT AdminPairingTableDelegate
+// TODO: Reimplement FabricTable to only have one backing store.
+class DLL_EXPORT FabricTableDelegate
 {
 public:
-    virtual ~AdminPairingTableDelegate() {}
+    virtual ~FabricTableDelegate() {}
     /**
-     * Gets called when an admin is deleted from KVS store.
+     * Gets called when an fabric is deleted from KVS store.
      **/
-    virtual void OnAdminDeletedFromStorage(AdminId adminId) = 0;
+    virtual void OnFabricDeletedFromStorage(FabricIndex fabricId) = 0;
 
     /**
-     * Gets called when an admin is loaded into Admin Pairing Table from KVS store.
+     * Gets called when an fabric is loaded into Fabric Table from KVS store.
```

Comment:
Watch-out for search-replace: `an admin` is now `an fabric`. Suggest auditing for `an fabric` and replacing with `a fabric`.